src/com/aca/homework/week5/light_out/Game.java

```diff
@@ -0,0 +1,155 @@
+package com.aca.homework.week5.light_out;
+
+import java.util.Scanner;
+
+public class Game {
+    private static int moves = 0;
+    private Light[][] board;
+
+    public Game() {
+        initialize();
+    }
+
+    private void initialize() {
+        boolean stateOff = false;
+        boolean stateOn = true;
+        board = new Light[5][5];
+        for (int i = 0; i < 5; i++) {
+            for (int j = 0; j < 5; j++) {
+                if (isStateOffLightsCoordinates(i, j)) {
+                    board[i][j] = new Light(stateOff);
```

Comment:
you could create a static factory of Light class with the name Light.ofOn() Light.ofOff()